package.json

```diff
@@ -4,9 +4,12 @@
 	"description": "Testing Sprint Challenge",
 	"main": "index.js",
 	"scripts": {
-		"test": "jest --watch --verbose",
+		"test": "cross-env DB_ENV=testing jest --watch --verbose",
```

Comment:
This _might_ be the line causing your DB to erase everything when running the tests. Do you need this?